rector.php

```diff
@@ -11,27 +11,21 @@
         __DIR__ . '/tests',
     ])
     ->withRootFiles()
-    ->withPHPStanConfigs([
-        __DIR__ . '/phpstan.dist.neon',
-        // rector does not load phpstan extension automatically so require them manually here:
-        __DIR__ . '/vendor/phpstan/phpstan-doctrine/extension.neon',
-        __DIR__ . '/vendor/phpstan/phpstan-symfony/extension.neon',
-    ])
-    ->withImportNames(importShortClasses: false)
-    ->withPreparedSets(codeQuality: true, doctrineCodeQuality: true)
-    ->withPhpSets()
 
-    // symfony rules
-    ->withSymfonyContainerPhp(__DIR__ . '/var/cache/website/dev/App_KernelDevDebugContainer.xml')
```

Comment:
This is also useful rather for Symfony 3 to 4 upgrade, especially with Symfony listener to subscriber migration.
 
The modern Symfony doesn't use any dynamic data. Can be dropped to make config more practical :+1:

Reply:
@TomasVotruba is it not required for analysing the container which is used a lot of times inside functional tests `static::getContainer()->get('doctrine');`.

Reply:
"is it not"... do you mean "it is"? 

Do you want to keep it?

These expression should be avoided and types should be used instead. E.g. 

```php
$entityManager = $this->get(EntityManagerInterface::class)
```

Reply:
While that is true for Project Services. Its not true for third party bundles like Sulu which still not expose the services over FCQN, we may change that in future but currently we still follow [symfony best practices](https://symfony.com/doc/7.1/bundles/best_practices.html#services) and expose service most service as `some_bundle.service_name`. We hope in future provide aliases for all common FCQN.

Reply:
Shall I keep it then?

Reply:
Yes would keep it at current state :)

Reply:
Restored :+1: